shared/pages/barbuddy.vue

```diff
@@ -49,7 +42,12 @@ const listBackgroundClass = runtimeConfig.public.domain == 'outsite.nl' ? 'bg-br
     </template>
 
     <div class="grid gap-4 md:grid-cols-2">
-      <div v-for="buddy in barBuddies" :key="buddy.name">
+      <div
+        v-for="buddy in barBuddies.filter(
+          (b) => b.sites.indexOf(runtimeConfig.public.domain) !== -1 && b[$i18n.locale]
+        )"
```

Comment:
```suggestion
        v-for="buddy in barBuddies.filter(
          (b) => b.sites.includes(runtimeConfig.public.domain) && b[$i18n.locale]
        )"
```

Reply:
Also, especially since we're using this twice, but also because it's quite a bit of logic for the template, I think it's a good idea to make this a [computed property](https://vuejs.org/guide/essentials/computed).